client/components/domains/registrant-extra-info/uk-form.jsx

```diff
@@ -137,7 +144,7 @@ export class RegistrantExtraInfoUkForm extends React.PureComponent {
 		const { ccTldDetails, translate } = this.props;
 		const registrationNumber = get( ccTldDetails, 'registrationNumber', '' );
 		const registrationNumberErrors = get(
-			this.props.validationErrors,
```

Comment:
I'm assuming you've verified that `validationErrors` is never passed to this component? This is the kind of thing that's probably worth mentioning in the commit message as context since the change isn't explicitly part of what the commit title describes.

Reply:
As far as I can tell no consumers of this component actually pass the `validationErrors` prop except in tests. I checked this by looking for any files that import the default export from `registrant-extra-info`. I have updated the PR description to make this more clear -- thanks for catching that.